File: Modul-4/soal-2/PRAK402_2410817210017_OrlandoSugian.c
#include<stdio.h>
int main()
{
int maks;
scanf("%d", &maks);
for (int i = 1; i <= maks; i += 2){
    printf("%d ", i);
}
printf("\n");
if (maks % 2 != 0){
    maks -= 1;
} 
for (int i = maks; i >=2; i -= 2){
    printf("%d ", i);
}return 0;
}

File: Modul-4/soal-2/PRAK402_2410817210017_OrlandoSugian.py
maks = int(input(""))
for i in range(1, maks + 1, 2):
    print(f"{i} ", end="")
print()
if maks % 2 != 0 :
    maks -= 1
for i in range(maks, 1, -2):
    print(f"{i} ", end="")